extensions/amp-skimlinks/0.1/skim-options.js

```diff
@@ -146,3 +164,14 @@ function getInternalDomains_(docInfo) {
 
   return internalDomains;
 }
+
+/**
+ * @param {!Element} element
+ */
```

Comment:
missing return annotation. `@return {!Object}`. getExcludedDomains_ is also missing `@return {!Array<string>}`